packages/framework/fluid-static/package.json

```diff
@@ -89,9 +71,11 @@
 		"prettier:fix": "prettier --write . --cache --ignore-path ../../../.prettierignore",
 		"test": "npm run test:mocha",
 		"test:coverage": "c8 npm test",
-		"test:mocha": "mocha --recursive \"dist/test/**/*.spec.*js\" -r node_modules/@fluid-internal/mocha-test-setup",
+		"test:mocha": "npm run test:mocha:cjs && npm run test:mocha:esm",
+		"test:mocha:cjs": "mocha  --recursive \"dist/test/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
+		"test:mocha:esm": "mocha  --recursive \"lib/test/*.spec.*js\" --exit -r node_modules/@fluid-internal/mocha-test-setup",
```

Comment:
nit: there are two spaces in between mocha and --recursive.

Reply:
Thx.  I fixed this in my template as well.